This PR proposes changes that would break the pipeline definition:
```
vertical-pod-autoscaler-enh_add-vpa-release-pipeline: Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/concourse/replicator.py", line 141, in render
    definition_descriptor = self._render(definition_descriptor)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/replicator.py", line 185, in _render
    'definition': factory.create_pipeline_definition(),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/factory.py", line 83, in create_pipeline_definition
    variant = self._create_variant(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/factory.py", line 138, in _create_variant
    variant._traits_dict = self._create_traits(raw_dict, variant_name)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/factory.py", line 189, in _create_traits
    traits_dict = {
                  ^
  File "/usr/lib/python3.11/site-packages/concourse/factory.py", line 190, in <dictcomp>
    name: TraitsFactory.create(
          ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/model/traits/__init__.py", line 68, in create
    raise ModelValidationError('no such trait: ' + str(name))
model.base.ModelValidationError: no such trait: git_tags

```

--- a/.ci/pipeline_definitions
+++ b/.ci/pipeline_definitions
@@ -137,6 +137,9 @@ vertical-pod-autoscaler:
           preprocess: 'finalize'
         release:
           nextversion: 'bump_minor'
+          git_tags:
+            - ref_template: refs/tags/vpa-{VERSION}
+          on_tag_conflict: 'fail'
         slack:
           default_channel: 'internal_scp_workspace'
           channel_cfgs:

